pkg/cedar/core/system_summary.go

```diff
@@ -67,6 +68,8 @@ func (c *Client) GetSystemSummary(ctx context.Context, opts ...systemSummaryPara
 	// Populate the SystemSummary field by converting each item in resp.Payload.SystemSummary
 	for _, sys := range resp.Payload.SystemSummary {
 		if sys.IctObjectID != nil {
+			uuid, _ := uuid.Parse(sys.UUID)
```

Comment:
nit - please check this error and maybe just log if it fails

Reply:
@samoddball addressed in https://github.com/CMSgov/easi-app/pull/2599/commits/a8ed8305ff335d3e3a8127ea4d46b2bec46f29b0

Reply:
thanks!